README.md

```diff
@@ -177,6 +177,39 @@ See [external_secrets](./ca_cdk_constructs/eks/external_secrets/external_secrets
 
 </details>
 
+### Storage
+
+<details>
+  <summary>AuroraFastClone</summary>
+
+Clones an Aurora cluster.
+
+```python
+clone = AuroraFastClone(self, "TestDBClone", source_cluster=aurora.cluster,
```

Comment:
Clones aren't replicas - so presumably we want a daily job to delete/re-clone? Assuming nothing changes though, we won't need to do the cdk equivalent of disabling the (e.g.) staging db stack, as there's nothing will normally change in the config day-to-day (unlike with snapshots where the latest available snapshot was updated multiple times per day)?

Reply:
correct, to "reseed" all we need to do is recreate the stack containing this construct